src/composables/storageRef.ts

```diff
@@ -1,9 +1,17 @@
 import { Ref, ref, watch } from 'vue';
 import type { Migration, StorageKey } from '@/types';
-import { asyncPipe } from '@/utils';
+import { asyncPipe, watchUntilTruthy } from '@/utils';
+import { IS_MOBILE_APP, STORAGE_KEYS } from '@/constants';
 import { WalletStorage } from '@/lib/WalletStorage';
+import migrateMnemonicToSecureStorage from '@/migrations/009-mnemonic-to-secure-storage';
```

Comment:
Migrations should be imported and applied in a file that defines the storageRef (accounts?).

Reply:
This migration will not run if it's applied in `accounts` because it needs to run earlier. I've changed it to be a function inside `storageRef` instead of a migration file as you are correct that it doesn't make sense like this